### PR TITLE
fixup: forgot canonical url

### PR DESCRIPTION
**Pull request by wommy**

### CODE REVIEW

**Comment by tannerdolby** [comment]
good catch! thanks @wommy

File: src/index.md [hunk 1]
```diff
@@ -242,7 +242,7 @@ options: `article`, `product`
 default: `website`
 
 ### url
-The content for `og:url` and `twitter:url`.
+The content for `<link rel=canonical href='URL'>`, `og:url` and `twitter:url`.
```
Comment (tannerdolby) [inline src/index.md]: Can you change `<link rel=canonical href='URL'>` to `<link rel="canonical" href="foo.com">` 

Once that is updated, this looks good to me.

**Review by tannerdolby — CHANGES_REQUESTED** [review]

File: src/index.md (hunk 1 above)
Comment (wommy) [reply src/index.md]: done

File: src/index.md [hunk 2]
```diff
@@ -242,7 +242,7 @@ options: `article`, `product`
 default: `website`
 
 ### url
-The content for `og:url` and `twitter:url`.
+The content for `<link rel=canonical href="foo.com">`, `og:url` and `twitter:url`.
```
Comment (tannerdolby) [inline src/index.md]: You forgot to address `rel=canonical` --> `rel="canonical"`

Once that's fixed this is ready to merge

**Review by tannerdolby — CHANGES_REQUESTED** [review]

File: src/index.md (hunk 2 above)
Comment (tannerdolby) [reply src/index.md]: 👋 @wommy Just a heads up, this is sort of a "missing piece" in the docs while waiting for this PR to be fixed up. 

I need this documentation to appear on the site so I'm going to close this and update it myself if not addressed by tomorrow. Thanks for your help thus far.

Comment (wommy) [reply src/index.md]: yo finished it!

i dont check everyday 🙇

**Review by tannerdolby — APPROVED** [review]
LGTM!

File: src/index.md (hunk 2 above)
Comment (tannerdolby) [reply src/index.md]: no worries! thanks for the fix

**Pull request merged**